support-frontend/app/controllers/Application.scala

```diff
@@ -150,7 +153,9 @@ class Application(
       paymentApiPayPalEndpoint = paymentAPIService.payPalCreatePaymentEndpoint,
       existingPaymentOptionsEndpoint = membersDataService.existingPaymentOptionsEndpoint,
       idUser = idUser,
-      guestAccountCreationToken = guestAccountCreationToken
+      guestAccountCreationToken = guestAccountCreationToken,
+      fontLoaderBundle = fontLoaderBundle,
```

Comment:
was this reordering intentional?

Reply:
Yes, the order now matches the class's definition.